clusters/data_spark_version.go

```diff
@@ -131,15 +131,20 @@ func DataSourceSparkVersion() *schema.Resource {
 
 		s["photon"].Deprecated = "Specify runtime_engine=\"PHOTON\" in the cluster configuration"
 		s["graviton"].Deprecated = "Not required anymore - it's automatically enabled on the Graviton-based node types"
+		common.AddWorkspaceIdField(s)
```

Comment:
this won't scale, add it on the level of `common.Resource` and `common.DataSource`

Reply:
I was thinking about how to do this. We don't know which resources are workspace-only or also at the account level. We need some explicit mapping to make sure this isn't forgotten about. I will probably add this at the provider-level to make sure there is an explicit opt-in/out for each resource type.